Doppler.UsersApi/Controllers/AccountController.cs

```diff
@@ -0,0 +1,36 @@
+using Doppler.UsersApi.DopplerSecurity;
+using Doppler.UsersApi.Infrastructure;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System.Threading.Tasks;
+
+namespace Doppler.UsersApi.Controllers
+{
+    [Authorize]
+    [ApiController]
+    public class AccountController
+    {
+        private readonly ILogger _logger;
+        private readonly IAccountRepository _accountRepository;
+
+        public AccountController(ILogger<FeatureController> logger, IAccountRepository accountRepository)
+        {
+            _logger = logger;
+            _accountRepository = accountRepository;
+        }
+
+        [HttpGet("/accounts/{email}/contactinformation")]
```

Comment:
Creo que el nombre tiene que ser `accountName` para que funcione con nuestra validación del token.

```suggestion
        [HttpGet("/accounts/{accountName}/contactinformation")]
```